website/api/zkvm/quickstart.md

```diff
@@ -5,96 +5,74 @@ slug: ./quickstart
 
 # zkVM Quick Start
 
-Welcome to the [zkVM] Quick Start page! Here are the steps between you and your first proof:
+Welcome to the zkVM Quick Start page! Here are the steps to create your first proof:
 
-1. Install the RISC Zero toolchain.
-2. Initialize a new project using the [cargo risczero] tool.
-3. Edit the boilerplate to build your first project.
-4. Run your project locally or with [Bonsai].
+1. Install the `risczero` toolchain
+2. Create a new project using the `risczero` tool.
+3. Familiarize yourself with the project's structure
+4. Run your project locally
+
+## Concept break
+
+zkVM provides a zero-knowledge proof of the correct execution of any Rust code.
+We want to run a piece of code and produce its proof of execution on zkVM. In the project code, we call zkVM a [prover]. The code we want to run on zkVM is a **[guest]** program. A proof of execution is a [receipt]; it contains a public part - a [journal] and an encryption part - a [seal].
+
+zkVM runs on the **[host]**, which loads the guest program to the zkVM and provides input to the guest. zkVM, or a prover, proves the guest execution by producing a locally verified receipt.
```

Comment:
This section seems a bit confusing to me. I think we should make it clear that there's the zkVM, in which the host is responsible for launching the zkVM, and then the guest is the program that runs inside of the zkVM.